src/datanode/src/datanode.rs

```
@@ -233,8 +234,18 @@ impl DatanodeBuilder {
             .new_region_server(log_store, region_event_listener)
             .await?;
 
-        self.initialize_region_server(&region_server, kv_backend, !controlled_by_metasrv)
-            .await?;
+        let region_server_moved = region_server.clone();
+
+        // Opens regions in background.
+        common_runtime::spawn_bg(async move {
```

Is the error be ignored?